frontend/src/router/pages/SubmitDates.jsx

```diff
@@ -0,0 +1,230 @@
+              </div>
+
+              <div className="d-none d-lg-block col-lg-1 text-center">
+                <span>&ndash;</span>
+              </div>
+
+              <div className="col-lg-5">
+                <div className="form-group">
+                  <label htmlFor="endDate1" className="form-label d-lg-none">
+                    End date
+                  </label>
+                  <input
+                    type="date"
+                    className="form-control"
+                    id="endDate1"
+                    name="endDate1"
+                  />
+                </div>
+              </div>
+            </div>
+
+            <p>
+              <button className="btn btn-text-primary">
+                + Add more operating dates
+              </button>
+            </p>
+          </div>
+
+          <div className="col-md-6">
+            <p>
+              Reservation dates{" "}
+              <FontAwesomeIcon
+                className="append-content"
+                icon="fa-solid fa-circle-info"
+              />
+            </p>
+
+            <p>Previous: Mon, Jun 7, 2025 - Wed, Sep 15, 2025</p>
+
+            <div className="row dates-row operating-dates">
+              <div className="col-lg-5">
+                <div className="form-group">
+                  <label htmlFor="startDate0" className="form-label d-lg-none">
+                    Start date
+                  </label>
+                  <input
+                    type="date"
+                    className="form-control"
+                    id="startDate0"
+                    name="startDate0"
+                  />
+                </div>
+              </div>
+
+              <div className="d-none d-lg-block col-lg-1 text-center">
+                <span>&ndash;</span>
+              </div>
+
+              <div className="col-lg-5">
+                <div className="form-group">
+                  <label htmlFor="endDate0" className="form-label d-lg-none">
+                    End date
+                  </label>
+                  <input
+                    type="date"
+                    className="form-control"
+                    id="endDate0"
+                    name="endDate0"
+                  />
+                </div>
+              </div>
+            </div>
+
+            <p>
+              <button className="btn btn-text-primary">
+                + Add more reservation dates
+              </button>
+            </p>
+          </div>
+        </div>
+      </section>
+    );
+  }
+
+  return (
+    <div className="page submit-dates">
+      <header className="page-header internal">
+        <h1>
+          Elk Falls Park #{parkId} / {seasonId} dates
+        </h1>
+      </header>
+
+      <h2>
+        <img src={groupCamping} className="sub-area-icon" /> Group Camping
+      </h2>
+
+      <p className="mb-5">
+        <a href="#TODO:-holidays-link">View a list of all statutory holidays</a>
+      </p>
+
```

Comment:
Just FYI, we use this API for the BC Parks: [https://canada-holidays.ca/api/v1/provinces/BC](https://canada-holidays.ca/api/v1/provinces/BC)

Reply:
Oh I guess it can be just a link.

Reply:
I'll have to see what they have in mind for that link. (If it pops up a modal window with our own list, or if it just links to some other page...) I'll leave it as a "TODO" for now but I don't want to forget it!